schema.sql

```diff
@@ -69,4 +69,13 @@ CREATE TABLE visits (
 	date_of_visit DATE,
 	FOREIGN KEY (vets_id) REFERENCES vets (id) ON DELETE CASCADE,
 	FOREIGN KEY (animals_id) REFERENCES animals (id) ON DELETE CASCADE
-);
\ No newline at end of file
+);
+
+-- FIFTH MILESTONE
+
+ALTER TABLE owners ADD COLUMN email VARCHAR(120);
+
+CREATE INDEX visits_animalsid_asc ON visits(animals_id ASC);
+DROP INDEX visits_animalsid_asc;
```

Comment:
- [ ] Please remove the drop index command because you suppose to only create the indexing that helps with query performance. 

- [ ] The execution time in the screenshots BEFORE is significantly lower than 1 sec (1000ms) and it's supposed to be higher than 1000 ms So please fix that.

[Hint] I suggest running the populate database query a few times more as described in the [project requirements](https://github.com/microverseinc/curriculum-databases/blob/main/db-structure/vet_clinic_performance_audit.md#:~:text=4%20Depening%20on,bigger%20than%201000ms.).